README.md

```diff
@@ -22,8 +20,7 @@ Requires:
   - xmltodict
```

Comment:
Now that we have significantly more requirements here, can we update this to just say we need Python 3.6 or higher, and packages as specified in the `requirements.txt`?

Reply:
i specified requirements.txt and changed it. it should be fixed now

Reply:
What I mean here is that in the `requirements.txt` file we  have  many more libraries than are mentioned here. Instead of listing every one in the README, let's just update this README file to say Python 3.6 or higher is required, as well as several supporting libraries as seen in requirements.txt.

Reply:
i changed it there